service/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation "org.databiosphere:workspacedataservice-client-okhttp-jakarta:0.2.115-20240227.195850-1"
     implementation "bio.terra:cbas-client:0.0.199"
     implementation "bio.terra:workspace-manager-client:0.254.1124-20240618.154106-1"
+    implementation project(':rawls-client')
```

Comment:
not your code, but while you're here could you add a space after `implementation` in line 54?